db/migrate/20160217192112_create_rentals.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRentals < ActiveRecord::Migration
+  def change
+    create_table :rentals do |t|
+      t.string :rental_status
+      t.integer :user_id, null: false
+      t.integer :department_id, null: false
+      t.integer :reservation_id, null: false
+      t.integer :fee_schedule_id, null: false
+      t.datetime :checked_out_at
+      t.datetime :checked_in_at
```

Comment:
Just to expound a bit, paper trail is a fantastic auditing tool. But using it to iterate different versions of the object and establish a invoice is comparatively slow / subject to failure in the event of a schema change.

To put it in context, if we use paper trail to establish when the customer checked in/out, we would have to walk the object and fetch the `updates_at` times when the object switches between states. That means we would have to detect the switch, record each time stamp, compare that against the scheduled check times. Doable, but not elegant.

We could always roll this (time stamps related to billing) into its own first class object.